test-shared/src/main/java/org/odk/collect/testshared/Interactions.kt

```diff
@@ -57,4 +59,14 @@ object Interactions {
         onView(view).perform(ViewActions.replaceText(text))
         closeSoftKeyboard()
     }
+
+    fun onListItem(listId: Int, position: Int, childViewId: Int): ViewInteraction {
```

Comment:
This doesn't really fit the pattern of the other helpers in `Interactions`/`Assertions`. Maybe instead you could pull out a custom helper (maybe to `ViewMatchers`) like `atPositionInRecyclerView(listId: Int, position: Int, childViewId: Int): Matcher<View` that you can then use with our helpers like `clickOn`:

```kotlin
clickOn(atPositionInRecyclerView(R.id.layers, 1, R.id.path))
```

Reply:
Done.